src/intervals/complex.jl

```diff
@@ -57,7 +56,7 @@ function sqrt_realpart(x::T, y::T,RND::RoundingMode) where T<:AbstractFloat
     ρ = ldexp(sqrt(ρ,RND),k) #sqrt((abs(z)+abs(x))/2) without over/underflow
 end
 
-function sqrt(z::Complex{Interval{T}}) where T<:AbstractFloat
+function sqrt(z::Complex{Interval{T}}) where {T<:AbstractFloat}
```

Comment:
Is there any reason to *not* define the method here as `z::Complex{Interval}`, similarly to other functions ?

Reply:
The only reason I saw was type the type parameter `T` was used in the body of this function. Of course we could use something in the form of `T = numtype(z)` instead.

Reply:
Fair enough... let's leave it as it is.